chia/rpc/full_node_rpc_client.py

```diff
@@ -159,6 +159,12 @@ async def get_coin_records_by_parent_ids(
         response = await self.fetch("get_coin_records_by_parent_ids", d)
         return [CoinRecord.from_json_dict(coin_record_dict_backwards_compat(coin)) for coin in response["coin_records"]]
 
+    async def get_aggsig_additional_data(self) -> str:
```

Comment:
Let's return bytes from this client endpoint.  `bytes32` if it's appropriate.

Reply:
Done